src/stirling/source_connectors/socket_tracer/uprobe_manager.cc

```diff
@@ -565,6 +580,12 @@ std::map<std::string, std::vector<int32_t>> ConvertPIDsListToMap(
     if (!fs::Exists(host_exe_path)) {
       continue;
     }
+    // Add filter here if the executable should be omitted
```

Comment:
Seems like this would be better suited to live outside this function (in `DeployGoUProbes` I believe), given the name of this function and its original purpose.

Reply:
I saw a [TODO](https://github.com/pixie-io/pixie/blob/962e48d742071a3c12f5fd731b103b351053b3af/src/stirling/source_connectors/socket_tracer/uprobe_manager.cc#L604-L605) that made me think there may have been a larger scope for this function, but I agree that it isn't the right fit.

I moved the filtering to the calling code in https://github.com/pixie-io/pixie/pull/1971/commits/afbaa9c3a188e887ebb5f636408c3256f974a4a4.